packages/twenty-front/src/modules/object-record/record-field/meta-types/input/hooks/useUpdateRelationFromManyFieldInput.tsx

```diff
@@ -0,0 +1,93 @@
+import { useContext } from 'react';
+import { useRecoilCallback } from 'recoil';
+
+import { useAttachRelatedRecordFromRecord } from '@/object-record/hooks/useAttachRelatedRecordFromRecord';
+import { useDetachRelatedRecordFromRecord } from '@/object-record/hooks/useDetachRelatedRecordFromRecord';
+import { FieldContext } from '@/object-record/record-field/contexts/FieldContext';
+import { objectRecordMultiSelectCheckedRecordsIdsComponentState } from '@/object-record/record-field/states/objectRecordMultiSelectCheckedRecordsIdsComponentState';
+import { assertFieldMetadata } from '@/object-record/record-field/types/guards/assertFieldMetadata';
+import { isFieldRelation } from '@/object-record/record-field/types/guards/isFieldRelation';
+import { FieldMetadataType } from '~/generated-metadata/graphql';
+
+export const useUpdateRelationFromManyFieldInput = ({
```

Comment:
I feel like it would be better to not talk about handlers in the naming of a hook, as it is always a local function naming relation to a component event, thus a hook that executes logic shouldn't be aware of where it is called.

Reply:
So maybe just rename the returned function instead